pyannote/audio/pipelines/speaker_verification.py

```diff
@@ -606,7 +606,7 @@ def __call__(
 
             embeddings[f] = self.session_.run(
                 output_names=["embs"],
-                input_feed={"feats": masked_feature.numpy()[None]},
+                input_feed={"feats": masked_feature.cpu().numpy()[None]},
```

Comment:
Here, you are actually sending the features back to CPU, before sending them to the inference. Therefore, I don't really understand why this should speed things up.

Can you please provide me with a bit more context? 
Ideally, a Colab notebook illustrating the speed up that this brings?

Reply:
If we don't use the cpu() method, it will result in an error. You can refer to the official PyTorch documentation for this. What I've observed is that if we don't place the tensor on the GPU, both torch.vstack() and torch.fft.rfft() will consume a significant amount of CPU resources.

```shell
# Under the 'wavs_16k' directory, there are a total of 8 '.lst' files, each of which contains paths to audio files. 
# After running the following code, 8 PyAnnote processes are launched on a computing node equipped 
# with 8 V100 GPUs, with each process occupying one GPU.
for list in `ls wavs_16k/*.lst`
do
    sleep 1s
    echo $list
    srun  -N1 -n1 --gres=gpu:1 python scripts/pyannote_main.py ${list} > logs/`basename $list .lst`_${time}.log 2>&1 &
done

```